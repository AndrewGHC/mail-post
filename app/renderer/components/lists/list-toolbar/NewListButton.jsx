import React from 'react';
import Button from 'material-ui/Button';
import AddIcon from 'material-ui-icons/Add';

const NewList = ({ ...props }) => (
  <Button
    type="submit"
    color="primary"
    data-test="import-csv"
    {...props}
  >
    <AddIcon />
    New List
  </Button>
);

export default NewList;
