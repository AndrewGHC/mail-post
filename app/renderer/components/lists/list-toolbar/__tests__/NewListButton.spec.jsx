import React from 'react';
import { shallow } from 'enzyme';

import NewListButton from '../NewListButton';

describe('NewListButton', () => {
  it('matches snapshot', async () => {
    const wrapper = shallow(<NewListButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
