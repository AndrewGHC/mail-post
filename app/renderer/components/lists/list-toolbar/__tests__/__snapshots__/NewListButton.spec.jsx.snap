// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewListButton matches snapshot 1`] = `
<WithStyles(Button)
  color="primary"
  data-test="import-csv"
  type="submit"
>
  <pure(Add) />
  Import
</WithStyles(Button)>
`;
